app/v1/views/answers.py

```diff
@@ -0,0 +1,44 @@
+# app/v1/views/answers.py
+
+from flask import Blueprint, jsonify, request
+
+from app.utilities import check_keys, check_empty_dict
+from app.v1.models import Answer, Question
+
+answers = Blueprint("answers_blueprint", __name__,
+                    url_prefix="/stackoverflowlite/api/v1/questions")
+
+answer_instance = Answer()
+question_instance = Question()
+
+
+@answers.route('/<question_id>/answers', methods=['POST'])
+def post_answer(question_id):
+    """method to post a new answer"""
+    answer_info = request.get_json()
+
+    # Check that all fields have been provided
```

Comment:
Checking if an answer has been provided and checking if empty should be abstracted into one validation function.

Reply:
This has been done.